### DISCUSSION TITLE
Rootfull Static IPv6 (--ip6) issue with multiple server apps in one container issue.

### DISCUSSION THREAD

**Discussion by Folaht** [containers/podman#13759] [Q&A]
Just like in my [other](https://github.com/containers/podman/discussions/13699) question, 
I try to start 16 server apps of the same kind in one container,  
one via run and 15 via exec.  
And only the first one is working.
So I have 1 server app running and 15 of the same kind that fail to connect.

Below is an example of such a failure:

`Error of node #1 (root node is #0)`
```
ERROR 2022-04-01T15:38:36.634678Z [sn/src/bin/sn_node.rs:L300]:
 ➤ Unfortunately we are unable to establish a connection to your machine 
([fdc2:9f3e:3d11:40c::1]:12001) 
either through a public IP address, or via IGD on your router. 
Please ensure that IGD is enabled on your router - 
if it is and you are still unable to add your node to the testnet, 
then skip adding a node for this testnet iteration. 
You can still use the testnet as a client, 
uploading and downloading content, etc. https://safenetforum.org/
```

[update]

I just realized I could also just start those 15 server apps in separate containers.

----

Using an RPi 4, steps to reproduce the issue:

1.
```
    sudo podman pull ghcr.io/safenetwork-community/rootnode-ipv6
```

2.
```
sudo podman run \
--name root_node \
--restart unless-stopped \
--publish [random.ip.v.6]:12000-12015:12000-12015/udp \
--env CON_IP=10.0.2.100 \
--env CON_PORT=12000 \
--env PUB_IP=my.ip.v.4 \
--env PUB_PORT=12000 \
--env VERBOSE=-vvvv \
--mount type=bind,source=/home/folaht/.local/share/safe/cli/,destination=/home/admin/.safe/cli/ \
-d ghcr.io/safenetwork-community/rootnode-ipv6:latest
```

3.
```
sudo podman exec root_node safe networks add folaht
sudo podman exec root_node safe networks switch folaht
```

    (This goes from 1 to 15, so port 12001 to 12015)

4.
```
sudo podman exec \
-d root_node sn_node -vvvv
--idle-timeout-msec 5500 \
--keep-alive-interval-msec 4000 \
--skip-auto-pont-forwarding \
--local-addr 10.2.0.100:12001 \
--public-addr [my.ip.v.6]:12001 \
--log-dr /home/admin/node_dir_1 \
--root-dir /home/admin/node_dir_1
```

**Comment by mheon** [comment]
Answering this without some knowledge of the application you're trying to run does not seem possible - what is the `safenetwork-community/rootnode-ipv6` server doing, how do the different server instances communicate, are the `podman exec` commands long-running and expected to manage the server invocations, and (perhaps most importantly) does it work as root?
